Formula/a/asdf.rb

```diff
@@ -32,7 +32,7 @@ def install
   def caveats
     <<~EOS
       To use asdf, add the following line (or equivalent) to your shell profile
-      e.g. ~/.profile or ~/.zshrc:
+      e.g. ~/.zprofile or ~/.zshrc:
```

Comment:
`~/.profile` is meant for shells that are _not_ Zsh, so `~/.profile` is actually correct.

Thanks for opening a PR anyway, though!